Frends.Salesforce.CreateSObject/Frends.Salesforce.CreateSObject/CreateSObject.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using static Frends.Salesforce.CreateSObject.Definitions.Enums;
+
+namespace Frends.Salesforce.CreateSObject
+{
+    /// <summary>
+    /// Tasks class.
+    /// </summary>
+    public class Salesforce
+    {
+        /// <summary>
+        /// Creates a sobject to Salesforce.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.Salesforce.CreateSObject)
+        /// </summary>
+        /// <param name="input">Information to create the sobject.</param>
+        /// <param name="options">Information about the salesforce destination.</param>
+        /// <returns>Object { object Body, bool RequestIsSuccessful, Exception ErrorException, string ErrorMessage }</returns>
+        public static async Task<Result> CreateSObject(
+            [PropertyTab] Input input,
+            [PropertyTab] Options options,
+            CancellationToken cancellationToken
+        )
+        {
+            if (string.IsNullOrEmpty(input.Domain)) throw new ArgumentNullException("Domain cannot be empty.");
+            else if (string.IsNullOrEmpty(input.SObjectAsJson)) throw new ArgumentNullException("Json cannot be empty.");
```

Comment:
You can just use if-statements instead of if-else, since all of these will throw an error if statement evaluates to true.